less/contact.less

```diff
@@ -0,0 +1,146 @@
+
+.contact {
+    .img-container {
+        width:100%;
+        position: relative;
+        color: white;
+
+        .contact-jumbotron {
+            width: 100%;
+            height: auto;
+
+            @media @mobile{
+                display: none;
+            }
+        }
+        .mobile-jumbotron {
+            display:none;
+
+            @media @mobile {
+                width: 100%;
+                display: block;
+            }
+        }
+        .inlayed-text {
+            font-size: 6.4rem;
+            font-weight: bold;
+            line-height: 7.5rem;
+            color: white;
+            position: absolute;
+            bottom: 16px;
+            left: 8%;
+
+            @media @mobile {
+                font-size: 3rem;
+                line-height: 4.2rem;
+                bottom: 5px;
+                letter-spacing: 2px;
+                }
+        }
+    }
+    .form-container {
+        margin: 0 auto;
+        padding-top:35px;
+        width: 90%;
+        display: flex;
+        justify-content: space-between;
+
+        @media @mobile {
+            flex-direction: column;
+        }
+
+        .contact-form {
+            display: flex;
+            flex-direction: column;
+            width: 50%;
```

Comment:
Amazing work on the stretch goal here by adding another page and adding in a form! 🥇